File: php/Program/Clothing.php
<?php

class Clothing extends Product {
    private $size = '';
    private $material = '';
    private $gender = '';

    public function __construct() {
    }

    public function setSize($size) {
        $this->size = $size;
    }

    public function getSize() {
        return $this->size;
    }

    public function setMaterial($material) {
        $this->material = $material;
    }

    public function getMaterial() {
        return $this->material;
    }

    public function setGender($gender) {
        $this->gender = $gender;
    }

    public function getGender() {
        return $this->gender;
    }

}

?>

File: python/Program/Clothing.py
from Product import Product

class Clothing (Product):
    def __init__(self):
        pass
        
    def getSize(self):
        return self.size
    
    def setSize(self, size):
        self.size = size
        
    def getMaterial(self):
        return self.material
    
    def setMaterial(self, material):
        self.material = material
        
    def getGender(self):
        return self.gender
    
    def setGender(self, gender):
        self.gender = gender